src/core/core/library_utils.cpp

```diff
@@ -71,7 +71,7 @@ namespace dsn {
             }            
 # elif defined(__linux__) || defined(__FreeBSD__) || defined(__APPLE__)
             module_name = "lib" + module_name + ".so";
-            auto hmod = dlopen(module_name.c_str(), RTLD_LAZY);
+            auto hmod = dlopen(module_name.c_str(), RTLD_LAZY|RTLD_GLOBAL);
```

Comment:
it seems the missing symbol is in Python, which we don't have control. According to http://stackoverflow.com/questions/480617/receive-undefined-symbol-error-when-loading-libary-with-dlopen, we probably should use global. Not 100 percent sure though:-) The dynamic link process of Linux just sucks~